src/main/NWN/API/Object/NwPlayer.cs

```diff
@@ -474,16 +484,6 @@ public void PopUpGUIPanel(GUIPanel panel = GUIPanel.Death)
     /// </summary>
     public void StopFade() => NWScript.StopFade(this);
 
-    /// <summary>
-    /// Forces this player to open the inventory of the specified placeable.
-    /// </summary>
-    /// <param name="target">The placeable inventory to be viewed.</param>
-    public void ForceOpenInventory(NwPlaceable target)
-    {
-      target.Placeable.m_bHasInventory = 1;
-      target.Placeable.OpenInventory(this);
-    }
-
```

Comment:
Flag this function as obsolete, so it won't break existing stuff. We can remove it in the next release.